src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -441,7 +431,15 @@
       <Link>Interop\Unix\System.Native\Interop.Write.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'net46'">
+  <ItemGroup Condition="'$(TargetGroup)'=='netcore50'">
+    <Compile Include="$(CommonPath)\Microsoft\Win32\SafeHandles\SafeHandleZeroOrMinusOneIsInvalid.cs">
+      <Link>Common\Microsoft\Win32\SafeHandles\SafeHandleZeroOrMinusOneIsInvalid.cs</Link>
+    </Compile>
```

Comment:
Should we even this Common code anymore?  I thought `SafeHandleZeroOrMinusOneIsInvalid` and friends was being added back to the public API surface (i.e. part of .NET Standard 2.0).  So, shouldn't these references just be coming from Microsoft.Win32.SafeHandles contract via a project.json reference